src/public/mathfield.ts

```diff
@@ -82,11 +82,13 @@ export type InsertOptions = {
 export interface Mathfield {
     mode: ParseMode;
 
-    getConfig<T extends keyof MathfieldConfig>(key: T): MathfieldConfig[T];
-    getConfig(keys: string[]): MathfieldConfig;
-    getConfig(keys: keyof MathfieldConfig | string[]): any | MathfieldConfig;
+    getConfig<K extends keyof MathfieldConfig>(
+        keys: K[]
+    ): Pick<MathfieldConfig, K>;
+    getConfig<K extends keyof MathfieldConfig>(key: K): MathfieldConfig[K];
+    getConfig(): MathfieldConfig;
 
-    $setConfig(config: MathfieldConfig): void;
+    $setConfig(config: Partial<MathfieldConfig>): void;
```

Comment:
This doesn't actually have an effect, because everything in the `MathfieldConfig` is optional, but at least it makes it a bit more explicit that you can pass in a partial object.

Reply:
Yeah, I like that. I'm wondering if `MathfieldConfig` shouldn't have optional fields, but instead just use `Partial<>` when appropriate. Seems like a clearer convention...

Reply:
That does sound like a reasonable idea. 👍